packages/create-wave-app/src/questions/index.ts

```diff
@@ -72,7 +72,7 @@ export const generateQuestions = [
       {
         name: "pnpm",
         value: "pnpm",
-        disabled: !isPnpmInstalled,
+        disabled: !isPnpmInstalled && "not installed",
```

Comment:
since "not installed" is a string won't it always be a truthy value?

Reply:
No, the text will be displayed in the command line if the test fails. it will also disable to option.